ideas42ventures/rails-react-starter
# feat: Rspec-OpenAPI + Rswag

# Fullstack PR Summary

[Asana Ticket](https://app.asana.com/0/1200734477019044/1201306410263605/f)

## Included Changes:

- Adds Rswag and rspec-openapi gems and the appropriate initializers and such.
- Also adds the generated documentation for the APIs in this repo.

## Notes to Reviewer








### New Environment Variables
RSWAG_USERNAME
RSWAG_PASSWORD

I added these to Heroku already and also added them to LastPass since you need it to login to the Heroku-launched app.

# How to QA
- Bundle install is absolutely necessary here.
- Launch the server as usual
- Go to <host>/api-docs
- Login with 'username' and 'password'
- You should see docs.
